projects/rero/ng-core/src/lib/record/editor/array-type/array-type.component.html

```diff
@@ -15,12 +15,12 @@
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 <!-- array title only if the array is empty -->
-<label *ngIf="to.label && to.hideLabel !== true && formControl && formControl.length === 0" [attr.for]="id">
+<label *ngIf="to.label && to.hideLabel !== true && formControl && formControl.length === 0" [attr.for]="id" tooltip="{{ to.description }}">
   <!-- add button only if the max is not reached -->
   <a *ngIf="canAdd()" (click)="add(0)" class="btn btn-link btn-sm">
     <i class="fa fa-plus"></i>
   </a>
-  <span [tooltip]="to.description">{{ to.label }}</span>
+  <span tooltip="{{to.description}}">{{ to.label }}</span>
```

Comment:
Why this change? Double moustache is the preferred way for binding attributes?

Reply:
I thought I was following the right syntax but both are possible.